1.6/Base/Source/BigSmallFramework/SimpleCustomRaces/RaceMorpher.cs

```diff
@@ -182,15 +189,26 @@ public static Pawn SwapAnimalToSapientVersion(this Pawn aniPawn)
                     }
                 }
 
+				// Remove hair color gene
+				GeneDef hairDef = newPawn.genes.GetHairColorGene();
+				Gene hairGene = newPawn.genes.Endogenes.FirstOrDefault(x => x.def == hairDef);
+				if (hairGene != null)
+					newPawn.genes.RemoveGene(hairGene);

```

Comment:
Make sure there is no hair gene on the final pawn to avoid any unexpected instant color changes.